services/bundlejs/bundlejs-package.service.js

```diff
@@ -0,0 +1,130 @@
+import Joi from 'joi'
+import { BaseJsonService } from '../index.js'
+
+const schema = Joi.object({
+  size: Joi.object({
+    compressedSize: Joi.string(),
+  }).required(),
+}).required()
+
+const keywords = ['node', 'bundlejs']
+
+const esbuild =
+  '<a href="https://github.com/evanw/esbuild" target="_blank" rel="noopener">esbuild</a>'
+const denoflate =
+  '<a href="https://github.com/hazae41/denoflate" target="_blank" rel="noopener">denoflate</a>'
+const bundlejs =
+  '<a href="https://bundlejs.com/" target="_blank" rel="noopener">bundlejs</a>'
+
+const documentation = `
+<p>
+ View ${esbuild} minified and ${denoflate} gzipped size of a package or selected exports, via ${bundlejs}.
+</p>
+`
+
+export default class BundlejsPackage extends BaseJsonService {
+  static category = 'size'
+
+  static route = {
+    base: 'bundlejs/package',
+    pattern: ':scope(@[^/]+)?/:packageName/:exports?',
+  }
+
+  static examples = [
+    {
+      title: 'npm package minimized gzipped size',
+      pattern: ':packageName',
+      namedParams: {
+        packageName: 'react',
+      },
+      staticPreview: this.render({ size: '2.94 kB' }),
+      keywords,
+      documentation,
+    },
```

Comment:
Will defer to Chris on whether we want to let this through as-is and refactor with the others later, or try to apply the new model now.

(we've changed the way we set some of these as part of an overhaul to the frontend site)

Reply:
I'm not really worried about this. Given the documentation still says to use `examples`, I think it is unreasonable to expect contributors to do something else as this point.